### PR TITLE
Added authors api endpoint and refactored views into a different package.

### PR DESCRIPTION
**Pull request by marek231**
## Description
Added a new endpoint /authors that retrieves a list of authors.
Refactored the views file into a views package that contains separate blueprints in separate files.

## Related
https://trello.com/c/K5xw3RBY/16-endpoint-to-retrive-list-of-authors

## Steps to Test or Reproduce

1. python3 run.py
2. curl 127.0.0.1:5000/authors/ (port number may differ)

### CODE REVIEW

**Comment by marek231** [comment]
[Endpoint to retrive list of authors](https://trello.com/c/K5xw3RBY/16-endpoint-to-retrive-list-of-authors)

File: api/schemas.py [hunk 1]
```diff
@@ -14,3 +14,16 @@ class Meta:
         self_view = 'books.get_by_id'
         self_view_kwargs = {'book_id': '<id>'}
         self_view_many = 'books.get_all'
+
+
+class AuthorSchema(Schema):
```
Comment (pirvudoru) [inline api/schemas.py]: Prefer the approach of separating classes into different files. It makes source navigation easier.

File: tests.py [hunk 2]
```diff
@@ -19,6 +19,14 @@ def test_all_books_api_mimetype(self):
         response = self.app.get('/books/')
         self.assertTrue(response.mimetype == 'application/vnd.api+json')
 
+    def test_all_authors_api_response(self):
+        response = self.app.get('/authors/')
+        self.assertEqual(response.status_code, 200)
+
+    def test_all_authors_api_mimetype(self):
+        response = self.app.get('/authors/')
+        self.assertTrue(response.mimetype == 'application/vnd.api+json')
```
Comment (pirvudoru) [inline tests.py:29]: Another good test would be to check the response.

File: run.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
 from api import create_app
 
 if __name__ == '__main__':
-    create_app().run(debug=False)
+    create_app().run(debug=True)
```
Comment (pirvudoru) [inline run.py:4]: Why we need to change this ?

File: api/views/books.py [hunk 4]
```diff
@@ -1,16 +1,7 @@
...
-def jsonify_schema_dump(*args, **kwargs):
-    """Wrapper around jsonify that sets the Content-Type of the response to
-    application/vnd.api+json.
-    """
-    response = jsonify(*args, **kwargs)
-    response.mimetype = 'application/vnd.api+json'
-    return response
-
+from flask import Blueprint
+from api.mock_db import db
```
Comment (pirvudoru) [inline api/views/books.py:2]: How can we abstract away the data source we are using? 
What about the serialization part?
Why would it be beneficial to do it?

File: run.py (hunk 3 above)
Comment (marek231) [reply run.py:4]: debug=True offers me debugger errors when accessing an endpoint. Otherwise it just says Internal Server Error.

File: tests.py [hunk 5]
```diff
@@ -1,5 +1,6 @@
 import unittest
 from api import create_app
+from api.mock_db import db
```
Comment (alexandru-calinoiu) [inline tests.py:3]: As you broke down views into different files, consider doing the same thing for tests.

**Pull request closed without merging**